.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         apt:
           - libopenjp2-7
       envs: |
-        - linux: py39
+        - linux: py310
```

Comment:
This is slow but when we merge in the affine transform PR, it will stop being slow.